src/edr_pydantic/my_base_model.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel as PydanticBaseModel
+from pydantic import ConfigDict
+
+
+class EDRBaseModel(PydanticBaseModel):
+    model_config = ConfigDict(
+        str_strip_whitespace=True,
+        extra="forbid",
```

Comment:
Also add `str_min_length=1` like in `covjson-pydantic`, or does that break stuff?

Reply:
It's in there, isn't it?